org.flatpak.easydict-gtk.yml

```diff
@@ -0,0 +1,58 @@
+app-id: org.flatpak.easydict-gtk
```

Comment:
Why it uses org.flatpak? It definitely not an app made by flatpak people

Reply:
Yeah, I was just about to say this - this absolutely doesn't pass the check in the review template:
>  I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).

flatpak.org does not have this policy, nor does the software come from flatpak.org. `one.jiri.easydict-gtk` would be a great app-id though!

Reply:
Thanks guys, I have thought about one.jiri, but I have thought that I can use org.flatpak. Sorry about it, it is my first flatpak package :).

Reply:
sorry I totally missed this.

Reply:
@hfiguiere but you have helped me a lot, so for me you are the star :D